.github/workflows/docker.build.yml

```diff
@@ -52,56 +49,54 @@ jobs:
           platforms: linux/amd64
           cache-from: type=local,src=/tmp/.buildx-cache
           cache-to: type=local,mode=max,dest=/tmp/.buildx-cache
-          push: ${{ github.event_name != 'pull_request' }}
           tags: ${{ steps.docker_meta.outputs.tags }}
           labels: ${{ steps.docker_meta.outputs.labels }}
-      #Staging: Create pull request on deploy repo from a merged PR
       -
-        name: create pull request for deploy on staging
+        name: Check out `wmde/wbaas-deploy` repository in staging child directory
         uses: actions/checkout@v3
-        # remove this if statement after fixing the PR
-        if: 1==2
         with:
           repository: wmde/wbaas-deploy
-      # Make changes to pull request here
-      - # TODO: Remove this if statement after fixing the PR
-        if: 1==2
+          path: ./repos/wbaas-deploy-staging
+      -
+        name: Check out `wmde/wbaas-deploy` repository in production child directory
+        uses: actions/checkout@v3
+        with:
+          repository: wmde/wbaas-deploy
+          path: ./repos/wbaas-deploy-production
+      - 
+        name: Update values for local, staging and production
+        id: update-values
         run: |
-          get_tag=$(echo ${{ steps.docker_meta.outputs.tags }} | cut -d':' -f2)
-          yq -i '.image.tag=$get_tag' k8s/helmfile/env/local/ui.values.yaml.gotmpl
-          yq -i '.image.tag=$get_tag' k8s/helmfile/env/staging/ui.values.yaml.gotmpl
+          TAG="$(echo ${{ steps.docker_meta.outputs.tags }} | cut -d':' -f2)"
+          echo "tag=$TAG" >> $GITHUB_OUTPUT
+          sed -i "/image:/{n;s/tag:.*/tag: $TAG/;}" ./repos/wbaas-deploy-staging/k8s/helmfile/env/local/ui.values.yaml.gotmpl
+          sed -i "/image:/{n;s/tag:.*/tag: $TAG/;}" ./repos/wbaas-deploy-staging/k8s/helmfile/env/staging/ui.values.yaml.gotmpl
+          sed -i "/image:/{n;s/tag:.*/tag: $TAG/;}" ./repos/wbaas-deploy-production/k8s/helmfile/env/production/ui.values.yaml.gotmpl
       - uses: peter-evans/create-pull-request@v4
-        # TODO: Remove this if statement after fixing the PR
-        if: 1==2
         with:
+          path: ./repos/wbaas-deploy-staging
           commit-message: 'Staging+Local: Deploy new UI image ${{ steps.docker_meta.outputs.tags }}'
```

Comment:
This is a very theoretical concern, but I wonder if we want to replace the "deploy" terminology in these PRs with "update" as no automated deployment is happening, it's more of an automated update a human would then deploy.

Reply:
Yeah, I can see us looking at making this more meaningful as time goes by. Right now I think it will do